db/redis.js

```diff
@@ -1,8 +1,8 @@
+require('dotenv').config();
 const Redis = require("ioredis");
 const radarData = require('./radarData.js');
 const doorData = require('./doorData.js');
 const stateData = require('./stateData.js');
-require('dotenv').config();
 const client = new Redis(6379, process.env.REDIS_CLUSTER_IP); // uses defaults unless given configuration object
```

Comment:
Is this value the same for production and test?
Do you not need to call the `getEnvVar` function here?

Reply:
Are you referring to `REDIS_CLUSTER_IP`? That's different in production and test, we use different env files when making the docker container for the respective deployments.

Reply:
Yes, that was the one that I was thinking of. And I think I see the difference now. It looks like it only matter when you have automated tests running the code and you don't have any tests yet that use this file. So maybe it doesn't matter.